src/components/pages/Home/RenderHomePage.js

```diff
@@ -1,18 +1,54 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import fetchCityData from '../../../api/cityData';
 import CityReport from '../../common/CityReport';
 import Title from '../../common/Title';
 import AddingCities from '../../common/AddingCities';
 import StaticHomePage from '../../common/StaticHomePage';
+import { SearchContext } from '../../../state/contexts/ReportContext';
+
 function RenderHomePage() {
+  // state of home page. default state is StaticHomePageComp
```

Comment:
Perfecto of moving this to a more state friendly location. RenderHomePage can just pass all the state the following components